### PR TITLE
implemented navbar and borders

### PR DESCRIPTION
**Pull request by ken12321**
Screen Shot 2021-08-08 at 10 03 32 pm

░░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄░░░░░░░
░░░░░█░░░░▒▒▒▒▒▒▒▒▒▒▒▒░░▀▀▄░░░░
░░░░█░░░▒▒▒▒▒▒░░░░░░░░▒▒▒░░█░░░
░░░█░░░░░░▄██▀▄▄░░░░░▄▄▄░░░░█░░
░▄▀▒▄▄▄▒░█▀▀▀▀▄▄█░░░██▄▄█░░░░█░
█░▒█▒▄░▀▄▄▄▀░░░░░░░░█░░░▒▒▒▒▒░█
█░▒█░█▀▄▄░░░░░█▀░░░░▀▄░░▄▀▀▀▄▒█
░█░▀▄░█▄░█▀▄▄░▀░▀▀░▄▄▀░░░░█░░█░
░░█░░░▀▄▀█▄▄░█▀▀▀▄▄▄▄▀▀█▀██░█░░
░░░█░░░░██░░▀█▄▄▄█▄▄█▄████░█░░░
░░░░█░░░░▀▀▄░█░░░█░█▀██████░█░░
░░░░░▀▄░░░░░▀▀▄▄▄█▄█▄█▄█▄▀░░█░░
░░░░░░░▀▄▄░▒▒▒▒░░░░░░░░░░▒░░░█░
░░░░░░░░░░▀▀▄▄░▒▒▒▒▒▒▒▒▒▒░░░░█░
░░░░░░░░░░░░░░▀▄▄▄▄▄░░░░░░░░█░░

### CODE REVIEW

File: my-app/src/components/SearchBar.css [hunk 1]
```diff
@@ -25,13 +34,43 @@
...
+  background-color: #75E6DA;
+  vertical-align: middle;
+  margin-left: 0.5%;
+  height: 3em;
+  width: 15%;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#navbar {
```
Comment (Andrewlaw171) [inline my-app/src/components/SearchBar.css]: This might look better centered thoughts?

**Review by Andrewlaw171 — APPROVED** [review]
Might be a good idea to centre the navbar contents apart from that looks good x

File: my-app/src/components/SearchBar.css (hunk 1 above)
Comment (ken12321) [reply my-app/src/components/SearchBar.css]: I agree, this was just boilerplate from some tutorial lol, will make an edit at some point to centre it before i merge

**Review by JinghanCode — APPROVED** [review]
Looks good x

**Comment by ken12321** [comment]
Alright, just pushed an update commit that will have centered the div, gonna merge to main now

**Pull request merged**